api/TextTrackList.json

```diff
@@ -79,10 +79,10 @@
               "version_added": null
             },
             "safari": {
-              "version_added": null
+              "version_added": "6"
```

Comment:
6 here but 7 below. Is that correct?

Reply:
Oddly enough, it is; `addTrack` was added before the other two events.

If you're talking about Safari iOS being 7, that's also correct; support wasn't implemented in iOS until iOS 7.0.

Reply:
Thanks for confirming!